boards/shields/wuxi_ov5640/doc/index.rst

```diff
@@ -0,0 +1,68 @@
+.. _wuxi_ov5640:
+
+WUXI_OV5640 Camera Module
+#########################
+
+Overview
+********
+
+This camera module connector comes from Wuxi A-KERR Science & Technology containing
+an OV5640 image sensor which provides the full functionality of a single chip 5
+megapixel (2592x1944) camera using OmniBSI™ technology in a small footprint package.
+
+More information about the OV5640 can be found at `OV5640 camera module`_.
+
+This module uses a 42-pin connector with MIPI CSI-2 interface which is available on
+NXP i.MX RT11XX series.
+
+Pins assignment of the Wuxi OV5640 camera module connector
+============================================================
```

Comment:
```suggestion
==========================================================
```

Underlines need to be the same length as the text above in RST

Reply:
yes, done